### DISCUSSION TITLE
Anchors for time dependent problems

### DISCUSSION THREAD

**Discussion by bakhtiyar-k** [lululxvi/deepxde#1618] [Q&A] [answered]
Hello,

If I want to use fixed anchor points for time-dependent problems with `num_domain=0`, do I need to provide them for each timestep? Let's say I have a point (x, y), I need to do (x, y, t0), (x, y, t_1), (x, y, t_2) ... (x, y, t_n)?

**Comment by lululxvi** [comment, marked as answer]
Yes, you need.
